ui/widgets/gridviewitem.go

```diff
@@ -175,7 +175,7 @@ func (g *GridViewItem) showContextMenu(pos fyne.Position) {
 			fyne.NewMenuItem("Shuffle", func() { g.onPlay(true) }),
 			fyne.NewMenuItem("Add to queue", g.onAddToQueue),
 			fyne.NewMenuItem("Add to playlist...", g.onAddToPlaylist),
-			fyne.NewMenuItem("Download...", g.onDownload)),
```

Comment:
I added the three dots here - it's meant to indicate the action will launch a dialog rather than immediately performing an action. Like this example https://help.gnome.org/users/gnumeric/stable/figures/menu-context-grid.png.en

I'm OK with removing them if it's confusing but I'll have to do it everywhere then for consistency

Reply:
Makes sense! There are some places where the Download option in the "meatbal" menu doesn't have the dots, for example, when you click a specific track to download. I can fix this in this PR to be consistent